CHANGELOG.md

```diff
@@ -4,24 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.9] - 2019-05-15
```

Comment:
This build gets submitted on Monday, 5/13, right? I think we should specify that date: the date the build is first available, rather than the date the build goes to 100% (which could change if we have to fix bugs).